From: =?utf-8?q?C=C3=A9dric_Boutillier?= <boutil@debian.org>
Date: Fri, 10 Jul 2020 22:38:11 +0200
Subject: relax minimal version for thor

Aruba builds fine with ruby-thor 0.20 and 1.x is not yet packaged.

Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=964618
Forwarded: not-needed
---
 aruba.gemspec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aruba.gemspec b/aruba.gemspec
index 1939381..e122d23 100644
--- a/aruba.gemspec
+++ b/aruba.gemspec
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'contracts', '~> 0.16.0'
   spec.add_runtime_dependency 'cucumber', ['>= 2.4', '< 6.0']
   spec.add_runtime_dependency 'rspec-expectations', '~> 3.4'
-  spec.add_runtime_dependency 'thor', '~> 1.0'
+  spec.add_runtime_dependency 'thor', '>= 0.20'
 
   spec.add_development_dependency 'json', '~> 2.1'
   spec.add_development_dependency 'license_finder', '~> 6.0'
